### PR TITLE
Add atreides03b

### PR DESCRIPTION
**Pull request by abcdefg30**

### CODE REVIEW

File: mods/d2k/maps/atreides-03b/atreides03b-AI.lua [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+SendAttack = function()
+	if Attacking then
+		return
+	end
+	Attacking = true
+	HoldProduction = true
+
+	local units = SetupAttackGroup()
+	Utils.Do(units, function(unit)
```
Comment (Mailaender) [inline mods/d2k/maps/atreides-03b/atreides03b-AI.lua]: Can be simplified to `Utils.Do(units, IdleHunt)`.

**Comment by Mailaender** [comment]
No surprises here. Works as promised. :white_check_mark:

**Comment by obrakmann** [comment]
Looks good to me, :+1: after the fixup

**Comment by abcdefg30** [comment]
Updated.

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/438ce844d5d9a6681f64e807777f4188b4e0bd61)

**Pull request merged**